config/src/config/mod.rs

```diff
@@ -409,12 +426,8 @@ mod test {
 
     #[test]
     fn verify_configs() {
-        let _ = vec![PUBLIC_FULL_NODE, VALIDATOR, VALIDATOR_FULL_NODE]
-            .iter()
-            .map(|path| {
-                NodeConfig::load_config(PathBuf::from(path))
-                    .unwrap_or_else(|e| panic!("Error in {}: {}", path, e))
-            })
-            .collect::<Vec<_>>();
+        NodeConfig::default_for_public_full_node();
+        NodeConfig::default_for_validator();
+        NodeConfig::default_for_validator_full_node();
```

Comment:
I guess this just tests that they load?

Reply:
yeah, let's just verify they load correctly...